### DISCUSSION TITLE
Use without <input> readonly

### DISCUSSION THREAD

**Discussion by Lennso** [dm4t2/vue-currency-input#350] [Q&A] [answered]
I use this great plugin for my calculator-application. The user can give some money input and i want to output the result with correct currency format. I try to build a component with an input-field and read-only html-attribute. It is working but the width of the input field has no fit to content. This is not very nice for designing.

It would be nice, if i can use vue-currency-input in a component to output the result in another element like a `<div></div>.` But with the ref="" it wouldn't work...

Working example:
`<input readonly ref="inputRef" type="text" />` --> input field with min-width --> not nice for styling

That would be nicer:
`<div>{{ value }}</div> `

Thanks for your help!

**Comment by dm4t2** [comment, marked as answer]
You can simply use [Intl.NumberFormat.prototype.format()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format) for printing currency formatted numbers.
